test/pom.xml

```diff
@@ -43,14 +43,21 @@
       <version>${wiremock.version}</version>
       <scope>provided</scope>
       <exclusions>
+    <!-- transitive dependency via wiremock -->
+    <!-- when wiremock is updated beyond 2.35, this can be removed -->
```

Comment:
This line is supposed to refer to json-smart, can you move it to that section of the POM?